README.rst

```diff
@@ -32,6 +32,16 @@ In order to use this library, you first need to go through the following steps:
 .. _Enable the Web Risk API.:  https://cloud.google.com/web-risk
 .. _Setup Authentication.: https://googleapis.dev/python/google-api-core/latest/auth.html
 
+Supported Python Versions
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Python >= 3.6
+
+Deprecated Python Versions
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Python == 2.7.
+
+The last version of this library compatible with Python 2.7 is google-cloud-webrisk==1.0.0 
```

Comment:
```suggestion
The last version of this library compatible with Python 2.7 is google-cloud-webrisk==0.3.0 
```

Reply:
curious that in setup.py, the release version is 1.0.0, which one should we take here? Thanks

Reply:
@xiaozhenliu-gg5 The microgenerator code has been merged and released (as 1.0.0). #25 The previous version was 0.3.0

Reply:
👍 sounds reasonable.